bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,1102 @@
+				<option value="4.5 s">4.5 Seconds</option>
+				<option value="6.5 s">6.5 Seconds (Slow)</option>
+				<option value="8.5 s">8.5 Seconds</option>
+				<option value="19 s">19 Seconds</option>
+				<option value="21.5 s">21.5 Seconds</option>
+				<option value="23.5 s">23.5 Seconds</option>
+				<option value="26 s">26 Seconds</option>
+				<option value="28 s">28 Seconds</option>
+				<option value="30 s">30 Seconds</option>
+				<option value="32 s">32 Seconds</option>
+				<option value="34 s">34 Seconds</option>
+				<option value="38.5 s">38.5 Seconds</option>
+				<option value="43 s">43 Seconds</option>
+				<option value="47 s">47 Seconds</option>
+				<option value="60 s">1 Minute</option>
+				<option value="90 s">1.5 Minutes</option>
+				<option value="120 s">2 Minutes</option>
+				<option value="150 s">2.5 Minutes</option>
+				<option value="180 s">3 Minutes</option>
+				<option value="210 s">3.5 Minutes</option>
+				<option value="240 s">4 Minutes</option>
+				<option value="270 s">4.5 minutes</option>
+				<option value="300 s">5 Minutes</option>
+				<option value="360 s">6 Minutes</option>
+				<option value="420 s">7 Minutes</option>
+				<option value="480 s">8 Minutes</option>
+				<option value="540 s">9 Minutes</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="relayMode" advanced="true">
+		<item-type>String</item-type>
+		<label>Relay Mode</label>
+		<description>Output Relay Mode</description>
+		<state>
+			<options>
+				<option value="LATCHING">Latching</option>
+				<option value="MOMENTARY_A">Momentary A</option>
+				<option value="MOMENTARY_B">Momentary B</option>
+				<option value="MOMENTARY_C">Momentary C</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="relaySensorFollow" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Relay Sensor Follow</label>
+		<description>Output Relay Follows Input Sensor</description>
+	</channel-type>
+
+	<channel-type id="reset">
+		<item-type>Switch</item-type>
+		<label>Reset</label>
+		<description>Reset (command only)</description>
+		<autoUpdatePolicy>veto</autoUpdatePolicy> <!-- command only -->
+	</channel-type>
+
+	<channel-type id="resumeDim" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Resume Dim</label>
+		<description>Resume at Last Dim Level</description>
+	</channel-type>
+
+	<channel-type id="reverseDirection" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Reverse Direction</label>
+		<description>Reverse Motor Direction</description>
+	</channel-type>
+
+	<channel-type id="rollershutter">
+		<item-type>Rollershutter</item-type>
+		<label>Rollershutter</label>
+		<description>Rollershutter</description>
+		<category>Rollershutter</category>
+		<autoUpdatePolicy>veto</autoUpdatePolicy> <!-- binding controls state updates -->
+	</channel-type>
+
+	<channel-type id="sceneOnOff">
+		<item-type>Switch</item-type>
+		<label>Scene On/Off</label>
```

Comment:
```suggestion
		<label>Scene Enabled</label>
```

or Active

Reply:
Please also adjust other channels in this file with `On/Off ` in its label.

Reply:
I renamed the `scene` thing type channels to `scene`, `fast-on-off`, `manual-change`.

All labels previously including `On/Off` have been renamed per your suggestion, except for `fast-on-off` since this is a specific Insteon function that is named that way and users would understand.